test_env_vars.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # Cachito API endpoint
-api_url: https://cachito.example.com/api/v1
```

Comment:
How's this set to a specific environment?

Reply:
It was in response for Matt's comment _I like option 2 but change the URL to point to http://localhost:8080/api/v1/ since that is the URL for the Cachito REST API in the docker-compose environment. Then you can use sed or some YAML editor to change the URL in the Jenkins job._ in https://gitlab.cee.redhat.com/exd-sp-cloud-build/cachito-tests/merge_requests/1

Reply:
Alright. That's fine. Instead of using `sed` to replace things, we could change the integration testing code itself to change the `api_url` based on an environment variable. Just brainstorming, no need to change anything for this PR.